src/components/site/tableOfContents/table-of-contents.module.scss

```diff
@@ -82,10 +86,10 @@
 
   :global {
     .md-list {
-      height: 100%;
-      max-height: calc(100vh - #{$pageNavHeight});
       padding-top: 0;
-      padding-bottom: $pageNavHeight;
+      position: absolute;
+      top: 270px;
+      width: 100%;
```

Comment:
So above you offset `.table-of-contents` by ~330px in order to get its child, `.md-list`, the el with overflow set to scroll, scroll over the desired vertical distance.  I think that 330px is not an accident, but is in fact the height of the toc + the height of the main header + the height of the page nav.  And so, I suspect, you can accomplish the same visual effect as you have now, just by setting the height of this el explicitly to the height of the screen (100vh) less the over fixed pos els (the height of the toc + the height of the main header + the height of the page nav), and removing the new positionoing stuff youo have on `.table-of-contents`.  So like `height: calc(100vh - 330px)` (326px on wide screens).  Or altogether 

```
.table-of-contents {
   z-index: 14;
}

.md-list {
   height: calc(100vh - 326px);
   padding-top: 0;
   padding-bottom: 0;

   @include respond($break80, max) {
      height: calc(100vh - 330px);
   }
}
```

I think....?

Reply:
After applying your proposed solution

Code:
![image](https://user-images.githubusercontent.com/8799443/108531569-105a1f00-7294-11eb-94c1-bb3fed97a2eb.png)


Firefox:
![image](https://user-images.githubusercontent.com/8799443/108531439-ebfe4280-7293-11eb-965d-97d7cf2ce882.png)

Chrome:
![image](https://user-images.githubusercontent.com/8799443/108531513-fcaeb880-7293-11eb-9860-acf95e5a6a39.png)

Reply:
That solution works great. Just need to keep this line in `.table-of-contents`
```
.table-of-contents {
  z-index: 14;
  padding-top: calc(#{$siteHeaderHeight} + 10px);  <----- this line
...
```
I also adjusted the height px values a little to account for the teeny-tiny space at the bottom.

Reply:
yes.  good catch.